src/rp2_common/boot_stage2/BUILD.bazel

```diff
@@ -0,0 +1,82 @@
+load("//bazel/toolchain:objcopy.bzl", "objcopy_to_bin")
+
+package(default_visibility = ["//visibility:private"])
+
+cc_library(
+    name = "config",
+    hdrs = [
+        "asminclude/boot2_helpers/exit_from_boot2.S",
+        "asminclude/boot2_helpers/read_flash_sreg.S",
+        "asminclude/boot2_helpers/wait_ssi_ready.S",
+        "boot2_at25sf128a.S",
+        "boot2_generic_03h.S",
+        "boot2_is25lp080.S",
+        "boot2_usb_blinky.S",
+        "boot2_w25q080.S",
+        "boot2_w25x10cl.S",
+        "include/boot_stage2/config.h",
+    ],
+    includes = ["include"],
```

Comment:
Shouldn't `asminclude` go in here instead of line 28?

Reply:
Yep, fixed.